source/features/infinite-commit-list-scroll.tsx

```diff
@@ -0,0 +1,92 @@
+import select from 'select-dom';
+import React from 'dom-chef';
+import fetchDom from '../libs/fetch-dom';
+import features from '../libs/features';
+
+// Used feature: infinite-scroll as template
+
+// Button from: https://primer.style/css/components/loaders
+const githubLoadingButton = <button className="btn mn-3" disabled><span>Loading</span><span className="AnimatedEllipsis"></span></button>;
+
+let container: HTMLElement;
+let link: HTMLAnchorElement | undefined;
+
+let lastDate: string;
+
+function getLastDate(document_?: HTMLDivElement): string {
+	if (!document_) {
+		document_ = select<HTMLDivElement>('.commits-listing')!;
+	}
+
+	const lastDivInCommitListing = select('.commits-listing > div:last-of-type', document_);
+	return lastDivInCommitListing!.textContent!;
+}
+
+async function appendOlder(): Promise<void> {
+	// Don't load if we don't have the link
+
+	if (!link?.href) {
+		return;
+	}
```

Comment:
This never happens or should never happen. This function is called by IntersectionObserver so `link` is being watched and it exists. If not, the observed should be disconnected earlier.